include/lsst/afw/image/Exposure.h

```diff
@@ -444,6 +444,24 @@ class Exposure {
      */
     Exposure getCutout(lsst::geom::Point2D const& center, lsst::geom::Extent2I const& size) const;
 
+    /**
+     * Return an Exposure that is a small cutout of the original.
+     *
+     * This is distinguished from subset() by allowing the box to extend off the image, and padding with
+     * NaN and mask NO_DATA in that case.
```

Comment:
I distinguish this from `subset` because `subset` also takes a box and returns a cutout, but raises instead of NaN-padding; the calling interface is the same, so I wanted to distinguish the use cases. The original `getCutout` only worked with coordinates on sky.

It's probably worthwhile adding the same sentence to `getCutout(Point, Extent)` as well, since that one is also similar to `subset`.